smart-contracts/wasm-chain-integration/benches/wasm.rs

```diff
@@ -326,7 +334,9 @@ pub fn criterion_benchmark(c: &mut Criterion) {
             group.bench_with_input(format!("write u8 n  = {} times as u32", n), n, |b, m| {
                 b.iter(|| {
                     assert!(
-                        artifact.run(&mut TestHost, "write_u32_u8", &[Value::I32(*m)]).is_ok(),
+                        artifact
+                            .run(&mut TestHost::uninitialized(), "write_u32_u8", &[Value::I32(*m)])
```

Comment:
Make sure to mark `TestHost::uninitialized` as const to not interfere with the benchmark.

The effect should be very small, likely negligible, but nevertheless we should be mindful.

Reply:
Done. Marked as `const`.